_drafts/_heat/using-ucd-ucdp.md

```diff
@@ -0,0 +1,74 @@
+
+**Notes:**
+
+* Only the default ports (8080 and 8443) are supported. If you want to use different ports, open a support ticket.
+* During installation, accept the default trial license path `27000@localhost` unless you have your own license. You can update the license after installation.
+* When prompted with the question "What host name will users access the web UI at?", if your host name is not registered, specify the IP address instead of the host name.
+* It is best practice to make the versions of the database server and the database driver consistent.
+
+For the complete UCDP documentation, see [IBM UCDP Knowledge Center](http://www-01.ibm.com/support/knowledgecenter/SSWS3W_6.1.1/com.ibm.edt.doc/edt61_welcome.html).
+
+## Installing UCD
+
+Follow these steps to install UCD:
+
+1. Install a database. Refer to the [Installing the database chapter](https://www-01.ibm.com/support/knowledgecenter/SS4GSP_6.1.1/com.ibm.udeploy.install.doc/topics/DBinstall.html) in the IBM UCD Knowledge Center.
+2. Install the server in the interactive mode. Refer to the [Install the server chapter](https://www-01.ibm.com/support/knowledgecenter/SS4GSP_6.1.1/com.ibm.udeploy.install.doc/topics/serverInstall.html) in the IBM UCD Knowledge Center.
+
+For the complete UCD documentation, see [IBM UCD Knowledge Center](https://www-01.ibm.com/support/knowledgecenter/SS4GSP_6.1.1/com.ibm.udeploy.doc/ucd61_welcome.html).
+
+**Note:**
+
+* Only the default ports (7918, 8080, and 8443) are supported. If you want to use different ports, open a support ticket.
+* It is best practice to make the versions of the database server and the database driver consistent.
+
+## Integrating UCD with UCDP
+
+Integrate your UCD with UCDP by following the instructions in [Connecting to IBM UrbanCode Deploy](http://www-01.ibm.com/support/knowledgecenter/SSWS3W_6.1.1/com.ibm.edt.doc/topics/integrate_ucd.html).
+
+## Connecting to IBM Cloud OpenStack Servers with UCDP and UCD
+
+1. For UCD and UCDP installed on external servers only: Use an OpenVPN client to connect to the cloud environment that you want to integrate with. For more information about accessing the environment, see [Setting up OpenVPN server on your instance](http://ibm-blue-box-help.github.io/help-documentation/gettingstarted/commontech/openvpn-setup/).
+
+2. Connect the UCDP server to IBM Blue Box by following the instructions in [Connecting to OpenStack-based clouds with Keystone authentication](http://www-01.ibm.com/support/knowledgecenter/SSWS3W_6.1.1/com.ibm.edt.doc/topics/cloud_connect_openstack_keystone.html).
+
+**Note:** In Step 2 Create an authentication realm that points to the Keystone server, keep the Use default orchestration engine check box checked when configuring realm.
```

Comment:
I would suggest rephrasing this to be:
When connecting the UCDP server to IBM Blue Box, create an authentication realm that points to the Keystone server, and keep the 'Use default orchestration engine check box' checked when configuring the realm.